src/style.css

```
@@ -1,3 +1,77 @@
+/* stylelint-disable selector-class-pattern */
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  /* background-color: yellow; */
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  margin: 0;
+  padding: 0;
+  line-height: 0;
+}
+
+.logo__img {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  height: 60px;
+  line-height: 60px;
+}
+
+.nav__list {
+  display: flex;
+  text-transform: uppercase;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  text-decoration: none;
+  font-size: 12px;
+  font-style: normal;
+}
+
+.navlinks {
+  color: #000;
+  text-decoration: none;
+  margin-left: 20px;
+  font-weight: 500;
+  padding: 23px 0;
+}
+
+[data-qa]:hover {
+  color: #00acdc;
+}
+
+.item:hover {
+  text-decoration: solid #00acdc;
```

```suggestion
  color: #00acdc;
```
you probably want to change the color on hover, right?

---

BTW, you don't use this class anywhere